apps/web/src/pages/templates/filter/FilterModal.tsx

```diff
@@ -304,30 +314,38 @@ function EqualityForm({
                   { value: 'SMALLER_EQUAL', label: 'Smaller or equal' },
                   { value: 'IN', label: 'Contains' },
                   { value: 'NOT_IN', label: 'Not contains' },
+                  { value: 'IS_DEFINED', label: 'isDefined' },
```

Comment:
```suggestion
                  { value: 'IS_DEFINED', label: 'Is Defined' },
```

Is this the value that is displayed to the user? Better to not do a camelCase :)